src/components/Body.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from 'styled-components';
+import Main from './Main';
+
+const AppBody = styled.div`
+    padding:40px 10px;
+`;
+
+function Body() {
```

Comment:
The way you declare the components in react is not good at all.

You have to show that a `function` and a `functional component` are indeed diff.
You should use this.

`const Body = (props) => ();` if its a single line return.

Reply:
Done